content/_partials/legacy/navbar.html.haml

```diff
@@ -34,7 +34,9 @@
           %li.leaf.last
             %a{:href => "https://reddit.com/r/jenkinsci", :title => "Latest Jenkins related news on /r/jenkinsci"}
               \/r/jenkinsci
-
+          %li.leaf.last
+            %a{:href => "https://wiki.jenkins-ci.org/display/JENKINS/Google+Summer+Of+Code+2016", :title => "Google Summer of Code 2016"}
+              Google Summer of Code 2016
```

Comment:
Agreed.

This link is now the last in the list, so `li.leaf.last` is correct — but you need to remove the `.last` from the Reddit link above.